drivers/can/can_native_linux.c

```diff
@@ -383,46 +384,46 @@ static int can_npl_recover(const struct device *dev, k_timeout_t timeout)
 }
 #endif /* CONFIG_CAN_AUTO_BUS_OFF_RECOVERY */
 
-static void can_npl_set_state_change_callback(const struct device *dev,
-					      can_state_change_callback_t cb,
-					      void *user_data)
+static void can_native_linux_set_state_change_callback(const struct device *dev,
```

Comment:
Just bike-shedding: Quite long names for static functions, good that we don't have the 80 chars limit anymore :)
(just ignore unless you really feel like changing it)